include/swift/SIL/SILInstruction.h

```diff
@@ -7771,23 +7771,25 @@ class GraphOperationInst final
   unsigned NumOperands;
   /// The attributes of the graph operation.
   MutableArrayRef<GraphOperationAttribute> Attributes;
+  /// Whether this instruction is to be executed out of graph (via eager op
+  /// dispatch). For an op with dynamic attribute(s), it must run out of graph.
+  bool RunOutOfGraph;
```

Comment:
At first glance this looks like "we ran out of graphs because we have too few of them". Maybe "noClustering"?

Reply:
Could someone please explain the meaning of "no clustering"? 😮

Reply:
Dan, the team recently converged on the user-facing term "auto-clustering" to describe our GPE technology (GPE is still the term we use to describe the underlying technology). 

Auto-clustering conveys the message that graph extraction is automatic, and is best-effort based (we no longer reject programs when we fail to GPE or deabstraction some parts of the program). 

Given a chunk of graph_ops, we try to "cluster" them into a single graph function, and that graph function can be viewed as a graph_op.

Reply:
nice!